### PR TITLE
{tools}[foss/2020a] IRkernel v1.1 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2602.swalot.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/17f3ec789761a630605094c1f7e965f8 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node308.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/c4e53008c286a44bb88b3c8908eba9d7 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 1 out of 2 (1 easyconfigs in this PR)
node127.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/5a98f93fa109049a6ed82f94704d5a68 for a full test report.

**Comment by boegel** [comment]
@lexming test failed because of pre-existing lock file?

File: easybuild/easyconfigs/i/IRkernel/IRkernel-1.1-foss-2020a-R-3.6.3-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    ('ZeroMQ', '4.3.2'),
+]
+
+exts_defaultclass = 'RPackage'
+exts_filter = ("R -q --no-save", "library(%(ext_name)s)")
+
+exts_default_options = {
+    'source_urls': [
+        'https://cran.r-project.org/src/contrib/',
+        'https://cran.rstudio.com/src/contrib/',
```
Comment (lexming) [inline easybuild/easyconfigs/i/IRkernel/IRkernel-1.1-foss-2020a-R-3.6.3-Python-3.8.2.eb:27]:
```suggestion
        'https://cran.rstudio.com/src/contrib/',
        'https://cran.r-project.org/src/contrib/Archive/IRkernel/',
```

**Review by lexming — CHANGES_REQUESTED** [review]
@boegel nope, my tests failed due to missing sources. Tarballs of v1.1 are now archived. Quick fix is to add the extra `source_url`. Alternatively the version of this easyconfig could be bumped to `1.1.1`.

File: easybuild/easyconfigs/i/IRkernel/IRkernel-1.1-foss-2020a-R-3.6.3-Python-3.8.2.eb (hunk 1 above)
Comment (verdurin) [reply easybuild/easyconfigs/i/IRkernel/IRkernel-1.1-foss-2020a-R-3.6.3-Python-3.8.2.eb:27]: @boegel any update here?

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="--force-download"

**Comment by boegel** [comment]
@lexming should be good to go now

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node381.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/8abbd0622c6e69398dc4c6041eb70784 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node101.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/26371a1da550619b97afc8550439b5c8 for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @boegel !

**Pull request merged**